pkg/credentialprovider/azure_credentials_test.go

```diff
@@ -76,89 +62,18 @@ func TestGetCredentials(t *testing.T) {
 	}
 }
 
-func TestGetCredentialsConfig(t *testing.T) {
```

Comment:
do we have enough ut coverage for this new track2 client?

Reply:
can you add back equivalent tests?

Reply:
These test has been covered in configloader package.

Reply:
ok.